examples/scenarioFlexiblePanel.py

```diff
@@ -0,0 +1,481 @@
+#
+
+r"""
+Overview
+--------
+
+This scenario demonstrates the capabilities of :ref:`spinningBodyNDOFStateEffector`, which represents an effector with
+N spin axes. The N-degree-of-freedom formulation allows the simulation of any number of attached rigid bodies with
+rotation capable about any axis. The spin axes and mass distribution of the rigid bodies are arbitrary.
+
+The scenario consists of a cylindrical hub with a flexible panel. The panel has two flexible directions: it can bend  or
+twist. It parameterized by the number of subsections, which means that the lumped-mass approach to the flexible dynamics
+can be refined with an increasing number of subpanels. The discretization into more subsections also allows for the
+modeling of more modes.
+
+The script is found in the folder ``basilisk/examples`` and executed by using::
+
+    python3 scenarioFlexiblePanel.py
+
+The scenario outputs six plots: the time history of bending angles, of torsional angles, of bending angle rates,
+of torsional angle rates, of the hub's attitude and of the hub's angular velocity. The scenario also creates a
+comprehensive Vizard simulation which creates an appropriate to-scale model for the defined scenario.
+
+Illustration of Simulation Results
+----------------------------------
+
+.. image:: /_images/Scenarios/scenarioFlexiblePanel_theta.svg
+   :align: center
+
+.. image:: /_images/Scenarios/scenarioFlexiblePanel_thetaDot.svg
+   :align: center
+
+.. image:: /_images/Scenarios/scenarioFlexiblePanel_beta.svg
+   :align: center
+
+.. image:: /_images/Scenarios/scenarioFlexiblePanel_betaDot.svg
+   :align: center
+
+.. image:: /_images/Scenarios/scenarioFlexiblePanel_sigma_BR.svg
+   :align: center
+
+.. image:: /_images/Scenarios/scenarioFlexiblePanel_omega_BR_B.svg
+   :align: center
+
+"""
+
+#
+# Basilisk Scenario Script and Integrated Test
+#
+# Purpose:  Illustrates the different simulation capabilities of the NDOF modules
+# Author:   João Vaz Carneiro
+# Creation Date:  May 15, 2024
+#
+
+import os
+import matplotlib.pyplot as plt
+import numpy as np
+
+from Basilisk.utilities import (SimulationBaseClass, vizSupport, simIncludeGravBody, macros, orbitalMotion,
+                                unitTestSupport, RigidBodyKinematics as rbk)
+from Basilisk.simulation import spacecraft, spinningBodyNDOFStateEffector, simpleNav, extForceTorque, svIntegrators
+from Basilisk.fswAlgorithms import mrpFeedback, inertial3D, attTrackingError
+from Basilisk.architecture import messaging
+
+from Basilisk import __path__
+
+bskPath = __path__[0]
+fileName = os.path.basename(os.path.splitext(__file__)[0])
+
+
+def run(show_plots, numberOfSegments):
+    scSim = createSimBaseClass()
+
+    scGeometry = geometryClass(numberOfSegments)
+    scSim, scObject, panel, extForceTorque = setUpDynModules(scSim, scGeometry)
+    scSim, thetaData, attErrorLog, _ = setUpFSWModules(scSim, scObject, panel, scGeometry, extForceTorque)
```

Comment:
The return message objects are never used.  As with the other script, we should remove these unused msg objects being return in the methods.

Reply:
Discussed above.